volumes/rescu/validation/identify_sensors_to_repair_i0617.sql

```diff
@@ -0,0 +1,123 @@
+--this query can be used to identify detectors for repair requests.
+    --recommend opening in qgis and styling based on 'classify'. Look for geographical gaps with bad/inactive sensors. 
+    --designate a set of `time_bins` (ie. current year) to get exact % active bins 
+    --includes inactive sensors that don't have any records during the designated `time_bins`
+--runs in 20s for 2023-01--2023-05
```

Comment:
orrrrrrrrrr... it could be a function that returns a SET/Table, with inputs of start and end date. I don't think this is necessary to implement rn, but a thought.